__tests__/data/af_router_output.json

```diff
@@ -58,7 +58,7 @@
           "app_version_name": "1.0.0",
           "bundleIdentifier": "com.rudderlabs.javascript",
           "customer_user_id": "12345",
-          "eventValue": "{\"user_actual_role\":\"system_admin, system_user\",\"user_actual_id\":12345}",
+          "eventValue": "{\"properties\":{\"user_actual_role\":\"system_admin, system_user\",\"user_actual_id\":12345}}",
```

Comment:
Properties is getting nested here is this expected?

Reply:
Please make sure the existing test cases are not having any breaking changes

Reply:
Yes, I changed this testcase only when I could see that earlier also properties was getting nested in `Order Completed` events. like [here](https://github.com/rudderlabs/rudder-transformer/blob/be5132d678576bbc9697492c78d494dd59ed3bf8/__tests__/data/af_output.json#L184)

Reply:
We are facing attribution issues in AppsFlyer because of this change.

Reply:
Hello @kishore-mega we would be happy to help you out cab we understand what is the issue that you are facing were you having event names in Uppercase and mapped to attribution as lowercase?

Reply:
Hi @utsabc ,
Let's continue the discussion regarding this issue [here](https://github.com/rudderlabs/rudder-transformer/issues/1451)